src/cmake/sentry.cmake

```diff
@@ -0,0 +1,66 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+# This CMAKE File will Integrate Sentry into MVPN
+
+
+# Defines which OS builds can include sentry. Check src/cmake Lists for all values of MVPN_PLATFORM_NAME
+set(SENTRY_SUPPORTED_OS  "windows" "macos")
```

Comment:
Why only these 2 platforms?

Reply:
Adding it is trivial, but i assume that I cannot use the `ExernalProject` with the PPA builds, as that would need to be present in the source bundle thing?